### PR TITLE
Remove repositories IDS-Serializer and IDS-CodeGeneration

### PR DESCRIPTION
**Pull request by heurtematte**
## Description 

These two repositories opened in 2022 got stale due to the decision in favor of another approach.
- https://github.com/eclipse-edc/IDS-Serializer
- https://github.com/eclipse-edc/IDS-CodeGeneration

Reasons: 
- no relation/usage from EDC
- no contributions from EDC community after initial contribution by IDSA
- source code was contributed by IDSA and still is available in their Org if any other project wants to use it
- no active users
- one doesn't even have code available 

## What this PR changes/adds

Remove repositories from otterdog configuration

## Linked Issue(s)
https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/4516

### CODE REVIEW

**Review by netomi — APPROVED** [review]

**Comment by netomi** [comment]
deleting IDS-CodeGeneration is certainly ok as its empty.
the second one could also be archived for now and deleted at a later time to avoid any surprises I guess.

**Comment by netomi** [comment]
@mspiekermann afaict, @heurtematte is waiting for an approval.

**Review by mspiekermann — APPROVED** [review]

**Pull request merged**
